src/config.json

```diff
@@ -10,5 +10,7 @@
   "RADIX_CLUSTER_TYPE": "development",
   "RADIX_ENVIRONMENT": "prod",
   "CLUSTER_EGRESS_IPS": "1.1.1.1,1.1.1.1",
-  "CLUSTER_INGRESS_IPS": "2.2.2.2,2.2.4.4"
+  "CLUSTER_INGRESS_IPS": "2.2.2.2,2.2.4.4",
+  "OAUTH2_CLIENT_ID": "fc082d3c-f792-4190-99f6-b63c1b270ea9",
+  "OAUTH2_AUTHORITY": "3aa4a235-b6e2-48d5-9195-7fcf05b459b0"
```

Comment:
Set OAUTH2_AUTHROITY to "https://login.microsoftonline.com/3aa4a235-b6e2-48d5-9195-7fcf05b459b0"